dbclient/MLFlowClient.py

```diff
@@ -11,13 +11,117 @@
 from thread_safe_writer import ThreadSafeWriter
 import concurrent
 from concurrent.futures import ThreadPoolExecutor
+import sqlite3
 
 class MLFlowClient:
     def __init__(self, configs, checkpoint_service):
         self._checkpoint_service = checkpoint_service
         self.export_dir = configs['export_dir']
         self.client = MlflowClient(f"databricks://{configs['profile']}")
 
+    def export_mlflow_runs(self, log_sql_file='mlflow_runs.db', experiment_log='mlflow_experiments.log', num_parallel=4):
+        """
+        Exports the Mlflow run objects. This can be run only after export_mlflow_experiments is complete.
+        Unlike other objects, we save the data into sqlite tables, given the possible scale of runs objects.
+        """
+        experiments_logfile = self.export_dir + experiment_log
+        mlflow_runs_checkpointer = self._checkpoint_service.get_checkpoint_key_set(
+            wmconstants.WM_EXPORT, wmconstants.MLFLOW_RUNS
+        )
+
+        start = timer()
+        con = sqlite3.connect(self.export_dir + log_sql_file)
+        with con:
+            con.execute('''
+              CREATE TABLE IF NOT EXISTS runs (id TEXT UNIQUE, start_time INT, run_obj TEXT)
```

Comment:
is the run id unique across experiments ?

Reply:
correct